manifests/l/LizardByte/Sunshine/0.14.1/LizardByte.Sunshine.locale.en-US.yaml

```diff
@@ -0,0 +1,35 @@
+# Created with YamlCreate.ps1 v2.1.3 $debug=QUSU.7-2-5
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.2.0.schema.json
+
+PackageIdentifier: LizardByte.Sunshine
+PackageVersion: 0.14.1
+PackageLocale: en-US
+Publisher: LizardByte
+PublisherUrl: https://app.lizardbyte.dev/
+PublisherSupportUrl: https://app.lizardbyte.dev/#Support
+# PrivacyUrl: 
+# Author: 
+PackageName: Sunshine
+PackageUrl: https://github.com/LizardByte/Sunshine
+License: GPL-3.0-only
+LicenseUrl: https://github.com/LizardByte/Sunshine/blob/master/LICENSE
+# Copyright: 
+# CopyrightUrl: 
+ShortDescription: Sunshine is a GameStream host for Moonlight. It is an open-source implementation of NVIDIA's GameStream, as used by the NVIDIA Shield
+# Description: 
+Moniker: sunshine
+Tags:
+- gamestream
+- gaming
+- gfe
+- moonlight
+- stream
+- streaming
+# Agreements: 
+# ReleaseNotes: 
+# ReleaseNotesUrl: 
+# PurchaseUrl: 
+# InstallationNotes: 
+# Documentations: 
```

Comment:
Is this meant to be a url?

You could use this `https://docs.lizardbyte.dev/projects/sunshine/en/latest`

Reply:
It seems to be a new entry from the new schema version, lemme check what it does.